fdbclient/ManagementAPI.actor.cpp

```diff
@@ -218,6 +218,22 @@ std::map<std::string, std::string> configForToken(std::string const& mode) {
 			out[p + key] = format("%d", mode);
 		}
 
+		if (key == "exclude") {
+			int p = 0;
+			while (p < value.size()) {
+				int end = value.find_first_of(',', p);
+				if (end == value.npos)
+					end = value.size();
+				auto addrRef = StringRef(value).substr(p, end - p);
+				AddressExclusion addr = AddressExclusion::parse(addrRef);
+				if (addr.isValid()) {
+					out[encodeExcludedServersKey(addr)] = "";
+				} else {
+					printf("Error: invalid address format: %s\n", addrRef.toString().c_str());
```

Comment:
Do we need to return here when addr is invalid?

Reply:
Good question. Thus function doesn't return an error, so return earlier is not very different from continuation in the loop: the map `out` is returned and applied as configuration changes. So I felt just applying the correct configs is fine.